### ISSUE TITLE
Unable to encode tiff file - No error message

### ISSUE THREAD

**Issue by barnaclebarnes** [uclouvain/openjpeg#812]
I'm trying to encode a `tiff` file and it fails with a not so useful error message:

```
➜  Downloads opj_compress -i 2009P_PNCC4-21-1-1956-N21_003038.tif -o 2009P_PNCC4-21-1-1956-N21_003038.jp2

[INFO] tile number 1 / 1
[ERROR] Cannot encode tile
failed to encode image: opj_encode
failed to encode image: opj_end_compress
failed to encode image
```

The file I am trying to encode is here:

https://www.dropbox.com/s/bi3fkgsap9fg9aa/2009P_PNCC4-21-1-1956-N21_003038.tif?dl=0

I'm using the latest 2.1.1 version of the software on a Mac installed using Homebrew and I also tries using the version built from source.

The image I am trying to encode is here:

https://www.dropbox.com/s/bi3fkgsap9fg9aa/2009P_PNCC4-21-1-1956-N21_003038.tif?dl=0

Any ideas of why this is not working? `kdu_compress` does compress the file.

**Comment by szukw000** [comment]
@barnaclebarnes,

the file seems to be too large:

image_to_j2k -i 2009P_PNCC4-21-1-1956-N21_003038.tif -o test.j2k

[ERROR] Error allocating memory for compressed bitstream
failed to encode image

winfried

**Comment by barnaclebarnes** [comment]
@szukw000 Yes - after many a google search I came up with something that did work:

`opj_compress -i 2009P_PNCC4-21-1-1956-N21_003038.tif -o 2009P_PNCC4-21-1-1956-N21_003038.jp2 -r 5 -n 7 -c "[256,256]"  -b "64,64" -p RPCL -SOP`

I guess the issue is that the error message for `opj_compress` is not correct. It should give the error allocating memory error.
